src/main/resources/com/google/api/codegen/go/discovery_fragment.snip

```diff
@@ -67,7 +67,7 @@ func main() {
         // {@TODO()} Handle error.
       }
     @else
-      err := {@methodPath(method)}({@callArg(method)}).Context(ctx).Do()
+      err = {@methodPath(method)}({@callArg(method)}).Context(ctx).Do()
```

Comment:
I changed it to break up the line length some, but it doesn't really save that many characters.
I've put it back.